networkx/generators/geometric.py

```diff
@@ -30,10 +31,52 @@ def euclidean(x, y):
 
 
 def geometric_edges(G, radius, p):
-    """Returns edge list of node pairs within `radius` of each other
+    """Returns edge list of node pairs within `radius` of each other.
```

Comment:
Do we put periods at the end of the one-liner descriptions? (obviously not important, but should be consistent).

Reply:
The example from the [numpydoc docstring style guide](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary) has a period, though this won't effect the docstring parsing in any way. I think it's up to us! If we want to have a policy on it, we can set one however we'd like.

Reply:
I think we usually leave off the period. But I checked the doc_strings in this module and in `generators` more broadly and I see a period is present for each of them.  So this is a good change. Sorry for the noise.
(and I Learned Something! :}